sdv/modeler.py

```diff
@@ -130,7 +130,7 @@ def _flatten_dict(cls, nested, prefix=''):
         for key in nested.keys():
```

Comment:
Please iterate over `items()` and avoid having to access `nested[key]` every time.